src/clarity.rs

```diff
@@ -76,6 +88,15 @@ macro_rules! panic_test {
     };
 }
 
+const HELIUM_BLOCK_LIMIT: ExecutionCost = ExecutionCost {
```

Comment:
nit: make var name consistent with `BLOCK_LIMIT_MAINNET` (so switch the order of the words in the variable name for one or the other)

Reply:
This is the same name that we use for this `ExecutionCost` instance in the `testnet/stacks-node` build target.  I was using it here for consistency.

More generally, I think we should fundamentally revisit the notion of having a different `ExecutionCost` on testnet.  I think the fact that they're different at all is a massive oversight -- by being more generous than mainnet, it's setting up developers for a nasty surprise when they deploy their code live.

Reply:
Can you move this const to `core::`, and then DRY the definition of `HELIUM_BLOCK_LIMIT` in the `stacks-node` config?

Reply:
Yup, done